lib/conjur/policy/resolver.rb

```diff
@@ -149,13 +149,18 @@ def resolve_owner record
         record.owner.id = absolute_path_of(record.owner.id) if record.owner && record.owner.id
       end
 
-      def resolve_grant record
+      def resolve_member record
         Array(record.member).each do |member|
-          member.role.id = absolute_path_of(member.role.id)
+          # Member has either role.id or id
```

Comment:
This comment should be expanded a bit more. It's not clear to me what each of these cases means.